systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -390,6 +391,32 @@ public static void waitForResourceReadiness(String resourceType, String resource
         LOGGER.info("Resource " + resourceType + "/" + resourceName + " is ready");
     }
 
+    public static <T extends CustomResource<? extends Spec, ? extends Status>> boolean waitForResourceStatusMessage(MixedOperation<T, ?, ?> operation, T resource, String message) {
+        long resourceTimeout = ResourceOperation.getTimeoutForResourceReadiness(resource.getKind());
+        return waitForResourceStatusMessage(operation, resource, message, resourceTimeout);
+    }
+
+    public static <T extends CustomResource<? extends Spec, ? extends Status>> boolean waitForResourceStatusMessage(MixedOperation<T, ?, ?> operation, T resource, String message, long resourceTimeout) {
+        waitForResourceStatusMessage(operation, resource.getKind(), resource.getMetadata().getNamespace(), resource.getMetadata().getName(), message, resourceTimeout);
+        return true;
+    }
+
+    public static <T extends CustomResource<? extends Spec, ? extends Status>> boolean waitForResourceStatusMessage(MixedOperation<T, ?, ?> operation, String kind, String namespace, String name, String message, long resourceTimeoutMs) {
+        LOGGER.info("Wait for {}: {} will contain desired status message: {}", kind, name, message);
+
+        TestUtils.waitFor(String.format("Wait for %s: %s will contain desired status message: %s", kind, name, message),
```

Comment:
This will double `wait for` string

Reply:
Yeah, you are right, good catch!

Reply:
Also removed from `waitForResourceStatus` `waitFor`